### PR TITLE
Documentation: Split apt-get example onto multiple lines

### PR DESCRIPTION
**Pull request by charleswhchan**
Following the best practice, reformat the documentation for RUN command.

> RUN
> As always, to make your Dockerfile more readable, understandable, and maintainable, put long or 
> complex RUN statements on multiple lines separated with backslashes.

### CODE REVIEW

**Comment by charleswhchan** [comment]
Do I need to sign the commit? I am simply reformatting the documentation ...

**Comment by jfrazelle** [comment]
yes you need to sign, thank you

File: docs/articles/dockerfile_best-practices.md [hunk 1]
```diff
@@ -153,9 +153,12 @@ maintainers. If you know there’s a particular package, `foo`, that needs to be
...
 * Do write instructions like:
-
-    RUN apt-get update && apt-get install -y package-bar package-foo package-baz
-
+```
+    RUN apt-get update && apt-get install -y \
+        package-bar \
+        package-foo \
+        package-baz
+```
```
Comment (thaJeztah) [inline docs/articles/dockerfile_best-practices.md]: Could you use "standard" Markdown here (i.e., just indentation, not the triple back-ticks)?

The tooling used for the documentation currently doesn't properly support GitHub-flavoured Markdown (it's on our list, but not there yet)

Comment (charleswhchan) [inline docs/articles/dockerfile_best-practices.md]: I tried tab indent but it didn't look right in GitHub preview. Ie. It turned into a single line with slashes. Would it render properly when the site is generated?

Comment (charleswhchan) [inline docs/articles/dockerfile_best-practices.md]: Updated patch to illustrate the issue:
https://github.com/charleswhchan/docker/blob/patch-2/docs/articles/dockerfile_best-practices.md

File: docs/articles/dockerfile_best-practices.md [hunk 2]
```diff
@@ -153,8 +153,10 @@ maintainers. If you know there’s a particular package, `foo`, that needs to be
 updated, use `apt-get install -y foo` and it will update automatically.
 
 * Do write instructions like:
-
-    RUN apt-get update && apt-get install -y package-bar package-foo package-baz
+    RUN apt-get update && apt-get install -y \
```
Comment (thaJeztah) [inline docs/articles/dockerfile_best-practices.md]: If you add a *blank line* before this line (to separate the code-block from the bullet), and indent the whole block with an additional 4 spaces (so, the first line is indented with 8 spaces), it should render correctly.

I just cloned your branch, made those changes and ran `make docs`. It then renders correctly;

14571-screenshot

**Comment by thaJeztah** [comment]
@charleswhchan I added a comment on how to solve the formatting, using just indentation / whitespace: https://github.com/docker/docker/pull/14571#discussion_r34542282

**Comment by charleswhchan** [comment]
@thaJeztah: thanks for the 'make docs' tip! I didn't know how to generate the pages before, and that is really neat.

I have reformat the code as suggested, only issue is that the 'package' is bold. Minor, I guess.

**Comment by SvenDowideit** [comment]
LGTM - I'm not bothered by the list of packages being non-alphabetical :)

if you do re-visit this before merge, I'd also put the `apt-get` command on a separate line, but I'm sure that bikeshed would have many different opinions :)

**Comment by charleswhchan** [comment]
@SvenDowideit, took care of the alphabetical ordering. :)

(Wasn't sure if you 2nd comment will create more disputes. So I let it alone for now.)

**Comment by thaJeztah** [comment]
Thanks for addressing those! The changes look good to me.

Next step; we prefer to have a single commit in stead of multiple. This means you'll have to "squash" your commits. 

You can find instructions on how to squash (and rebase) on the [work on your issue](https://docs.docker.com/project/work-issue/) page of the documentation (and lots of other information in that section, such as the `make docs`). Feel free to ask for assistance in the #docker-dev IRC channel if you run into problems there.

I realize this is a lot of work for a "small" change, but we're obviously hoping you'll be making more contributions after this, so see it as a learning experience :smile:

Thanks so much for your work so far!

**Comment by charleswhchan** [comment]
All commits squash into 1; recommit into the branch. (Let me know if I didn't do it properly.)

**Comment by thaJeztah** [comment]
LOL, erm, looks like you lost the "alphabetic ordering" change in the process.

You can fix that and `git commit -s --amend` to amend the commit and force push.

Let me know if you're willing to fix that, otherwise I'm okay with merging it as-is :-)

**Comment by charleswhchan** [comment]
:( Fixed. Sorry for the trouble.

**Comment by thaJeztah** [comment]
LGTM! Thanks so much, and hoping to see more contributions from you now that you're familiar with the process (sorry, we don't have "certificates" for "certified docker contributor", otherwise I would have given you one :smile:)

**Comment by charleswhchan** [comment]
Thanks! :)

**Pull request merged**